odoo/addons/base/models/ir_ui_view.py

```diff
@@ -506,6 +507,7 @@ def create(self, vals_list):
 
         self.env.registry.clear_cache('templates')
         result = super(IrUiView, self.with_context(ir_ui_view_partial_validation=True)).create(vals_list)
+        result._check_xml()
```

Comment:
ir_ui_view_partial_validation should be used after the creation since it is only used by the `_check_xml`.
```suggestion
        result.with_context(ir_ui_view_partial_validation=True)._check_xml()
```